script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -42,7 +42,7 @@ As she says that, I let out an 'Ah' as everything slides into place.
 [sha:2f94cd978caa190bc82c289c8bd47007f46eb9d6]{
 -- Page 47, Offset 1274.
 -- この先輩はいつだって、誰かのために<奔走|ほんそう>している人なのだ。
-Senpai is the kind of person who's always working hard for someone else.
+Senpai's the type that's always working hard for other people's sakes.
```

Comment:
internet seems [torn on this](https://english.stackexchange.com/questions/61567/for-both-our-sake-or-for-both-our-sakes) but ngram seems to agree with me more or less.
> other people's sake